openapi-module/src/GPT/config.js

```diff
@@ -0,0 +1,188 @@
+module.exports = {
+    useGPT: false,  // set this to true to use GPT for generating scenarios
```

Comment:
Instead of hardcoding value to set the flag, use an argument in the command, eg `--useGPT=true|false`

Reply:
Have you done any changes regarding this comment?

Reply:
yes, integrated this into the run commands

Reply:
Update the README where needed, there are still references mentioning that you should go and edit the config.js manually to enable the feature